### DISCUSSION TITLE
Feedback about the "private profile" options

### DISCUSSION THREAD

**Discussion by giroletm** [community/community#50204] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi there, I've got some feedback to give about private profile options.

In short, it'd be great to have more options, rather than a binary "make profile private" option with no customization possible.
The options I have in mind would be for private profiles to be able to:
- Display badges
- Display activity
- Display achievements
- Display followers (which would allow people to follow you even if your profile is private)
- Display followings

However, I'm sure that a lot of other options would be nice, basically anything that's hidden by private profiles should have their individual checkbox to be displayed anyway if the account owner wants to.

What do you think about it?

**Comment by vronskyvitaly** [comment]
Great suggestion, that would be great!
